src/Migrate/Migrate.Autorest/custom/Initialize-AzMigrateReplicationInfrastructure.ps1

```diff
@@ -13,6 +13,21 @@
 # limitations under the License.
 # ----------------------------------------------------------------------------------
 
+function Create-RoleAssignments(
+    [string]$objectId,
+    [string]$saId) {
+
+    $storageBlobDataContributorRoleDefinitionId = "ba92f5b4-2d11-453d-a403-e96b0029c9fe"
```

Comment:
It's a hardcoded id. Could you explain more about this?

Reply:
This is id of the role definition (Storage Blob Data Contributor), taken from the azure documentation: [link](https://learn.microsoft.com/en-us/azure/role-based-access-control/built-in-roles#:~:text=ba92f5b4%2D2d11%2D453d%2Da403%2De96b0029c9fe)

Reply:
I see the ids are defined in 'src/Migrate/Migrate.Autorest/custom/Helper/AzStackHCICommonSettings.ps1'. Please refer to this ps1 file like [this](https://github.com/Azure/azure-powershell/blob/21d2c2b48f65de2fd8810f91a4f3a18e94154eaf/src/Migrate/Migrate.Autorest/custom/Get-AzMigrateHCIServerReplication.ps1#L134).